pkg/operator/apis/monitoring/v1alpha1/types.go

```diff
@@ -1156,6 +1156,36 @@ type ClusterRulesList struct {
 	Items           []ClusterRules `json:"items"`
 }
 
+// GlobalRules defines Prometheus alerting and recording rules that are scoped
+// to all data in the queried project. Only metric data from the current cluster is processed
```

Comment:
The godoc needs to be updated; this is a description of `ClusterRules` behavior.

Reply:
No :) it is adapted.

Reply:
Ah, actually forgot to delete one old sentence. Fixed.

Reply:
So I'm a bit hung up on
```
Only metric data from the current cluster is processed
```
Shouldn't this be something like
```
Only metric data from the queried project is processed
```

Reply:
Yes, that's the old sentence I meant. Didn't push anymore yesterday after the presubmit run.